src/doc/src/reference/unstable.md

```diff
@@ -1233,63 +1233,147 @@ cargo check -Z unstable-options -Z check-cfg-well-known-values
-keywords = ["cli"]
-categories = ["development-tools"]
-publish = false
-edition = "2018"
 ```
 
 ```toml
-# in a workspace member's Cargo.toml
+# [PROGECT_DIR]/bar/Cargo.toml
 [package]
+name = "bar"
 version.workspace = true
 authors.workspace = true
 description.workspace = true
 documentation.workspace = true
-readme.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
-license-file.workspace = true
-keywords.workspace = true
-categories.workspace = true
-publish.workspace = true
 ```
 
+
+### The `workspace.dependencies` table
+
+The `workspace.dependencies` table is where you define dependencies to be
+inherited by members of a workspace. Inheriting a dependency is as simple as
+[specifying the dependency][specifying-dependencies] in this table then
+[inheriting that dependency][inheriting-a-dependency-from-a-workspace] in the member
+`[dependencies]` table.
+
+Dependencies from this table can be referenced by `[dependencies]`, 
+`[dev-dependencies]`, `[build-dependencies]`, and `[target."...".dependencies]`.
+Dependencies from this table cannot be declared as `optional`.`features` declared
+in this table are additive with the `features` from `[dependencies]`, see [features]
+
+Full example:
+```toml
+# [PROGECT_DIR]/Cargo.toml
+[workspace]
+memebers = ["bar"]
+
+[workspace.dependencies]
+dep = { version = "0.1", features = ["fancy"] }
+dep-build = "0.8"
+dep-dev = "0.5.2"
+```
+
+```toml
+# [PROGECT_DIR]/bar/Cargo.toml
+[project]
+name = "bar"
+version = "0.2.0"
+
+[dependencies]
+dep = { workspace = true, features = ["dancy"] }
+
+[build-dependencies]
+dep-build.workspace = true
+
+[dev-dependencies]
+dep-dev.workspace = true
+```
+
+[inheriting-a-dependency-from-a-workspace]: #inheriting-a-dependency-from-a-workspace
+[workspace-metadata-table]: workspaces.md#the-workspacemetadata-table
+[workspaces]: workspaces.md
+
+
+### Inheriting a dependency from a workspace
+
+*Stabilization*: This would be in [`specifying-dependencies.md`][specifying-dependencies],
+under [Renaming dependencies in Cargo.toml][renaming-dependencies-in-cargotoml]
+
+Dependencies can be inherited from a workspace. To do this, specify the
+dependency in the workspace's [`[workspace.dependencies]`][workspace.dependencies] table.
+After that add it to the `[dependencies]` table with `dep.workspace = true`.
+
+
+ The `workspace` key cannot be defined with:
+
+| A - P              | Q - Z            | 
+|--------------------|------------------|
+| `branch`           | `registry`       |
+| `default-features` | `registry-index` |
+| `git`              | `rev`            |
+| `package`          | `tag`            |
+| `path`             | `version`        |
+
+The `workspace` key can be defined with:
+- [`optional`][optional] -  Note that the`[workspace.dependencies]` table is not allowed to specify `optional`.
+- [`features`][features] - These are additive with the features declared in the `[workspace.dependencies]`
```

Comment:
Unless there is a cargo style thing I've not noticed, then I'd recommend `:` rather than `-` for separating out an item from the content for it.